node/src/actors/chain_manager/actor.rs

```diff
@@ -252,7 +249,11 @@ impl ChainManager {
                         warn!("Checkpoint zero is in the future ({:?}). Delaying chain bootstrapping until then.", date);
                         // Subscribe to first epoch
                         epoch_manager_addr
-                            .do_send(Subscribe::to_epoch(0, chain_manager_addr, EpochPayload))
+                            .do_send(Subscribe::to_epoch(0, chain_manager_addr.clone(), EveryEpochPayload));
```

Comment:
Is going to trigger the same handler as `Subscribe::to_all`? If so, it will be executed twice for epoch 0. Or does `Subscribe::to_all` only send messages when the epoch is >= 1?

Reply:
The second

Reply:
Or not? Mmm...

Reply:
Yes, it was redundant. I removed the redundant parts, thanks!!